src/Dfc.CourseDirectory.WebV2/Features/Home/HomeController.cs

```diff
@@ -28,10 +28,9 @@ public IActionResult Index([FromServices] ICurrentUserProvider currentUserProvid
             }
         }
 
-        [HttpGet("help")]
         [HttpGet("Home/Help", Order = 99)]
         [AllowDeactivatedProvider]
-        public IActionResult Help() => View();
+        public IActionResult Help() => RedirectToAction("Index", "Contact");
```

Comment:
Pondering if we should do this sort of thing in middleware?

Reply:
I think this is fine

Reply:
`Order` parameter is no longer required, though

Reply:
Good spot. Fixed.

Reply:
Unrelated to this but we should maybe consider adding `[AllowDeactivatedProvider]` to `ContractController.Index` too

Reply:
Closely related enough. Fixed.